final_prep/mod_zero_hero.rb

```diff
@@ -2,51 +2,95 @@
 
 # Declare two variables - hero_name AND special_ability - set to strings
 
+hero_name = "ManBat"
+special_ability = "looking really cool"
+
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
+greeting = "Hello my name is #{hero_name}."
+catchphrase = "Spoooooonnnnn!"
```

Comment:
🚨 uh oh, the instructions were "catchphrase should be assigned to a string that uses interpolation to include the special_ability"

Don't get me wrong. It is an excellent catchphrase